src/components/customer-resource-show.js

```diff
@@ -8,8 +8,10 @@ import KeyValueLabel from './key-value-label';
 import IdentifiersList from './identifiers-list';
 import ContributorsList from './contributors-list';
 import ToggleSwitch from './toggle-switch';
+import CoverageDates from './coverage-dates';
+import { isBookPublicationType, formatPublicationType, isValidCoverageList } from './utilities';
 
-export default function CustomerResourceShow({ customerResource, toggleRequest, toggleSelected }) {
+export default function CustomerResourceShow({ customerResource, toggleRequest, toggleSelected }, { intl }) {
```

Comment:
why is `intl` separated here?

Reply:
I was following the same usage pattern as existed with package-show.js. It looks like the separation is due to intl being a contextType vs a propType (as other parameters)